packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -610,7 +610,10 @@ export class EditorPage {
 	 * @param {boolean} visit Whether to then visit the page.
 	 * @returns {URL} Published article's URL.
 	 */
-	async publish( { visit = false }: { visit?: boolean } = {} ): Promise< URL > {
+	async publish( {
+		visit = false,
+		timeout,
+	}: { visit?: boolean; timeout?: number } = {} ): Promise< URL > {
```

Comment:
Not strictly necessary, but useful change to have so callers can override the default timeout. This is really useful for the Donations block on AT.